packages/backend-modules/publikator/lib/File/utils.ts

```diff
@@ -0,0 +1,54 @@
+const {
+  lib: { s3 },
+} = require('@orbiting/backend-modules-assets')
+const { slugify } = require('@orbiting/backend-modules-utils')
+
+import { RepoFileRow } from '../../loaders/File'
+
+const { ASSETS_SERVER_BASE_URL, AWS_S3_BUCKET } = process.env
+
+export function getKey(file: RepoFileRow): string {
+  return ['repos', file.repoId, 'files', file.id, file.name].join('/')
+}
+
+export function getSafeFileName(name: string): string {
+  const safeName = slugify(name)
+
+  if (!safeName?.length) {
```

Comment:
shouldn't we default to something (e.g. 'file_' + timestamp) instead of throwing?

Reply:
Gets hard to recognize which file was which when you upload multiple files. Thus failing is intentional.